thinc/backends/cupy_ops.py

```diff
@@ -108,15 +108,15 @@ def frobenius_norm(X):
         gradient *= cupy.minimum(threshold, grad_norm) / grad_norm
         return gradient
 
-    def seq2col(self, seq, nW):
+    def seq2col(self, seq, nW, lens=None):
         """Given an (M, N) sequence of vectors, return an (M, N*(nW*2+1)) sequence.
         The new sequence is constructed by concatenating nW preceding and succeeding
         vectors onto each column in the sequence, to extract a window of features.
         """
-        return _custom_kernels.seq2col(seq, nW)
+        return _custom_kernels.seq2col(seq, nW, lens=lens)
```

Comment:
Internally it can be named how we like. Probably `lengths` everywhere is better though.

```suggestion
        return _custom_kernels.seq2col(seq, nW, lens=lengths)
```